### PR TITLE
Init challenge

### PR DESCRIPTION
**Pull request by awoznica**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+*       @ipartalas @awoznica @mtiteux
```
Comment (ioannispartalas) [inline CODEOWNERS:1]: We should fix the issue here.

Comment (awoznica) [reply CODEOWNERS:1]: what should be in this file?

File: README.md [hunk 2]
```diff
@@ -1 +1,173 @@
...
+propose the next clicked hotel. The baseline and evaluation metric are implemented in Python and Tensorflow.
+
+In order to run the baseline you can use the following command:
+```sh
+python baseline_evaluator.py --train_path path_to_train_file --eval_path path_to_test_file
+```
+
+## Software requirements
+
+The code has been tested with `Python 3.8` and requires `Tensorflow>=2.0`, `pandas>=1.4`, `numpy>=1.18` and
```
Comment (ioannispartalas) [inline README.md:110]: Here is TF 2.7.1 we have in the requirements

Comment (awoznica) [reply README.md:110]: 2.7.1>=2.0 so it should be ok ?

**Review by ioannispartalas — APPROVED** [review]

**Pull request merged**